CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 [1]: https://pypi.org/project/google-cloud-pubsub/#history
 
+## [1.3.0](https://www.github.com/googleapis/python-pubsub/compare/v1.2.0...v1.3.0) (2020-02-05)
+
+
+### Features
+
+* **pubsub:** ordering keys ([#26](https://www.github.com/googleapis/python-pubsub/issues/26)) ([cc3093a](https://www.github.com/googleapis/python-pubsub/commit/cc3093a2c0304259bc374bc2eeec9630e4a11a5e))
```

Comment:
The `**pubsub:**` prefix is redundant here, and release notes should contain full sentences (AFAIK).

Reply:
I talked to Ben (who made the bot) and he says it just comes from the "pubsub" part of "feat(pubsub)". The part between the parens just provides extra context. We can omit it unless we want to standardize on a few contextual tags: tests, system-tests, gapic, publish-path, subscribe-path, etc. Just a thought.

I'm going to merge the PR and then send a follow-up to clean up these unnecessary "pubsub" tags.